tests/unit/cli/admin/test_ceremony.py

```diff
@@ -554,7 +554,7 @@ def test_ceremony_option_save(
         ]
         assert ceremony._run_ceremony_steps.calls == [pretend.call(True)]
         assert ceremony.save_payload.calls == [
-            pretend.call("payload.json", {"k": "v"})
+            pretend.call("payload.json", {"k": "v", "timeout": 300})
         ]
 
     def test_ceremony_option_save_OSError(
```

Comment:
Instead of using an old test and adding the timeout in the payload what is better to do is to add a separate test maybe called `test_ceremony_option_timeout` and use that.
As an example, we have a separate test for the `save` option starting on line 518.